src/test/ui/destructuring-assignment/tuple_struct_destructure_fail.stderr

```diff
@@ -15,22 +15,26 @@ LL |     Enum::SingleVariant(a, .., b, ..) = Enum::SingleVariant(0, 1);
    |                            previously used here
 
 error[E0023]: this pattern has 3 fields, but the corresponding tuple struct has 2 fields
-  --> $DIR/tuple_struct_destructure_fail.rs:30:5
+  --> $DIR/tuple_struct_destructure_fail.rs:30:16
```

Comment:
What's causing this change?

Reply:
Huh, actually the old span seems incorrect and the new one is right. I'm very confused...

Reply:
Oops, I just realized I was misinterpreting the column number as a line number—that explains it.